CDCgov/prime-simplereport
# Old pending org schema clean up in verify org view Part 1/2

# FRONTEND PULL REQUEST

## Related Issue

- Partially implements #5786 

## Changes Proposed

- Removed all checks for org types using the old graphql schema in edit/verify org page.

## Additional Information

- This is only the frontend clean up and a backend ticket will come after.
- I was able to verify that no orgs with the old schema are ever returned to the frontend by checking [PR 3989](https://github.com/CDCgov/prime-simplereport/pull/3989).

## Testing
Code is available in Dev1
- Sign up for a new org
- Edit the org data
- Verify the org
